Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiStripGainsAAG_cff.py

```diff
@@ -98,7 +98,6 @@
                                             # 2 provide more detailed output
                                             Frequency = cms.untracked.int32(50),
                                             MEPathToSave = cms.untracked.string('AlCaReco/SiStripGainsAAG'),
-                                            deleteAfterCopy = cms.untracked.bool(True)
```

Comment:
@tocheng @christopheralanwest @mmusich please suggest how to test this ALCAPROMPT, using ```--dasquery=file dataset=/ZeroBias/Run2016C-SiStripCalMinBiasAfterAbortGap-18Apr2017-v1/ALCARECO run=276097``` in a similar way as done for SiStrpGains does not work:
```
Begin processing the 1st record. Run 276092, Event 32587086, LumiSection 74 on stream 0 at 22-Nov-2019 10:43:57.009 CET
----- Begin Fatal Exception 22-Nov-2019 10:43:57 CET-----------------------
An exception of category 'Configuration' occurred while
   [0] Processing  Event run: 276092 lumi: 74 event: 32587086 stream: 0
   [1] Running path 'pathALCARECOPromptCalibProdSiStripGainsAAG'
   [2] Calling method for module HLTHighLevel/'ALCARECOCalMinBiasFilterForSiStripGainsAAG'
Exception Message:
requested HLT path "pathALCARECOSiStripCalMinBiasAAG" does not exist
----- End Fatal Exception -------------------------------------------------
```

Reply:
Did you change the ALCA producer?

Reply:
```
cmsDriver.py step3 --datatier ALCARECO --conditions auto:run2_data -s ALCA:PromptCalibProdSiStripGainsAAG --eventcontent ALCARECO -n 1000 --dasquery=file dataset=/ZeroBias/Run2016C-SiStripCalMinBiasAfterAbortGap-18Apr2017-v1/ALCARECO run=276097 ```

Reply:
@fabiocos, the command is correct but the input dataset is not.  
In 2017 we changed 'SiStripCalMinBiasAfterAbortGap' to 'SiStripCalMinBiasAAG' due to limitations in the output dataset name length in dataset storage system.
Because of this the PD you chose is not processable tout-court in recent releases, but needs to have this line https://github.com/cms-sw/cmssw/blob/master/Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiStripGainsAAG_cff.py#L9 adjusted.
To make it work out of the box just switch to a more recent dataset like
 '/StreamExpress/Run2018C-SiStripCalMinBiasAAG-Express-v1/ALCARECO' (is at 'T2_CH_CERN')

Reply:
@mmusich with ```step3 --processName ALCA2 --datatier ALCARECO --conditions auto:run2_data -s ALCA:PromptCalibProdSiStripGainsAAG --eventcontent ALCARECO -n 1000 --dasquery=file dataset=/StreamExpress/Run2018C-SiStripCalMinBiasAAG-Express-v1/ALCARECO``` I get

![stripaag](https://user-images.githubusercontent.com/4058194/69449283-d9c4ea00-0d5a-11ea-9892-4bdb8fb1290d.png)

which looks to me comparable to your test at the file crossing boundaries